salesforce/Ballerina.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-distribution = "2201.3.0"
+distribution = "2201.7.0"
 org = "ballerinax"
 name = "salesforce"
 version = "7.2.0"
```

Comment:
We need to bump to the next minor version with the distribution version update. But this version is still outdated, I think the latest released version is `7.4.0`. But why it is not updated here?

Reply:
Lemme update the version,